src/screens/Categories/screens/AddCategory/AddCategory.tsx

```diff
@@ -1,11 +1,10 @@
 import { Formik } from 'formik';
 import React from 'react';
-import AddCategoryForm from './components/AddCategoryForm';
+import CategoryForm from './components/CategoryForm';
 import { addCategory } from '../../../../services/categoryService';
 import { useNavigation } from '@react-navigation/native';
```

Comment:
better practice chai package imports haru top ma rakhne ani module imports haru tala ani tyo ni ladder structure bhaye best. 
Ani constants ko euta section, components ko euta, utils ko euta teseri, ek ek line ko gap for all